samples/java_springboot/21.corebot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/BookingDialog.java

```diff
@@ -23,8 +23,8 @@
  * The class containing the booking dialogs.
  */
 public class BookingDialog extends CancelAndHelpDialog {
-    private final String destinationStepMsgText = "Where would you like to travel to?";
-    private final String originStepMsgText = "Where are you traveling from?";
+    private static String destinationStepMsgText = "Where would you like to travel to?";
+    private static String originStepMsgText = "Where are you traveling from?";
```

Comment:
Rollback this as the feedback in Slite is wrong.

C# props are [const](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/21.corebot-app-insights/Dialogs/BookingDialog.cs#L15-L16) which means that in Java should be `final` as it was.